### PR TITLE
Bind input enums' `NONE` values

### PR DESCRIPTION
**Pull request by piiertho**
This adds enum values bind:
- `Key::NONE`
- `MouseButton::NONE`
- `MIDIMessage::NONE`

This fixes inconsistencies in api.  
In new `extension_api.json` you can see `Key::NONE` is not exposed. But there are methods that uses it as default value, like `PopupMenu::add_item`.  
This leads to not being able to retreive the value for `Key` enum matching `0` (the default value).

### CODE REVIEW

**Comment by akien-mga** [comment]
You will need to add documentation for the num enum values (generate the template with `godot --doctool`), see https://docs.godotengine.org/en/latest/community/contributing/updating_the_class_reference.html

**Comment by piiertho** [comment]
> You will need to add documentation for the num enum values (generate the template with `godot --doctool`), see https://docs.godotengine.org/en/latest/community/contributing/updating_the_class_reference.html

done :)

File: doc/classes/@GlobalScope.xml [hunk 1]
```diff
@@ -1273,6 +1273,9 @@
 		<constant name="INLINE_ALIGNMENT_TEXT_MASK" value="12" enum="InlineAlignment">
 			A bit mask for [code]INLINE_ALIGNMENT_TO_*[/code] alignment constants.
 		</constant>
+		<constant name="KEY_NONE" value="0" enum="Key">
+			Enum value when no key.
```
Comment (akien-mga) [inline doc/classes/@GlobalScope.xml:1277]: That sounds pretty awkward.
How about:
```suggestion
			Enum value which doesn't correspond to any key. This is used to initialize [Key] properties with a generic state.
```
Same for others.

Comment (piiertho) [reply doc/classes/@GlobalScope.xml:1277]: Far better, will change this

File: doc/classes/@GlobalScope.xml [hunk 2]
```diff
@@ -2173,6 +2179,9 @@
 		<constant name="JOY_AXIS_MAX" value="10" enum="JoyAxis">
 			The maximum number of game controller axes: OpenVR supports up to 5 Joysticks making a total of 10 axes.
 		</constant>
+		<constant name="MIDI_MESSAGE_NONE" value="0" enum="MIDIMessage">
+			Enum value which doesn't correspond to any midi message. This is used to initialize [enum MidiMessage] properties with a generic state.
```
Comment (akien-mga) [inline doc/classes/@GlobalScope.xml:2183]:
```suggestion
			Enum value which doesn't correspond to any MIDI message. This is used to initialize [enum MidiMessage] properties with a generic state.
```

**Review by akien-mga — APPROVED** [review]

**Pull request merged**